java/code/src/com/redhat/rhn/frontend/dto/PackageDto.java

```diff
@@ -621,4 +622,19 @@ public String getFile() {
     public void setExtraTags(Map<String, String> extraTagsIn) {
         this.extraTags = extraTagsIn;
     }
+
+    /**
+     * @return retracted
+     */
+    public Boolean getRetracted() {
```

Comment:
I think we should return `boolean` and name the function `isRetracted()`

Reply:
Hm, I think for boxed booleans, the convention is `getXY`. Perhaps @lucidd knows more.

Reply:
i usually let intelij generate getters and setters and it seems to use `get` for `Boolean` and `is` for `boolean`. Generally i would trust intelij to do whatever the common convention is in that case.